e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -8,28 +8,30 @@
 // ***************************************************************
 
 describe('Plugin Marketplace', () => {
-    after(() => {
-        // # Restore default configuration
-        const newSettings = {
-            PluginSettings: {
-                Enable: true,
-                EnableMarketplace: true,
-                MarketplaceUrl: 'https://api.integrations.mattermost.com',
-            },
-        };
-        cy.apiUpdateConfig(newSettings);
-    });
+    // after(() => {
+    //     // # Restore default configuration
+    //     const newSettings = {
+    //         PluginSettings: {
+    //             Enable: true,
+    //             EnableMarketplace: true,
+    //             MarketplaceUrl: 'https://api.integrations.mattermost.com',
+    //         },
+    //     };
+    //     cy.apiUpdateConfig(newSettings);
+    // });
```

Comment:
👍

Reply:
Oh wait, let me check this again on why did I commented did out...

Reply:
Reverted the change.  Although the `after` hook / reverting state is not necessary as we do setups during `before`